bin/tachyon-workers.sh

```diff
@@ -22,9 +22,15 @@ TACHYON_LIBEXEC_DIR=${TACHYON_LIBEXEC_DIR:-$DEFAULT_LIBEXEC_DIR}
 HOSTLIST=$TACHYON_CONF_DIR/workers
 
 for worker in `cat "$HOSTLIST"|sed  "s/#.*$//;/^$/d"`; do
-  echo -n "Connection to $worker as $USER... "
-  ssh -o ConnectTimeout=5 -o StrictHostKeyChecking=no -t $worker $LAUNCHER $"${@// /\\ }" 2>&1
-  sleep 0.02
+  if [ -n "${TACHYON_SSH_FOREGROUND}" ]; then
+    echo -n "Connection to $worker as $USER... "
+    ssh -o ConnectTimeout=5 -o StrictHostKeyChecking=no -t $worker $LAUNCHER $"${@// /\\ }" 2>&1
+  else
+    ssh -o ConnectTimeout=5 -o StrictHostKeyChecking=no -t $worker $LAUNCHER $"${@// /\\ }" 2>&1 &
+  fi
+  if [ "$TACHYON_SLAVE_SLEEP" != "" ]; then
```

Comment:
Okay, in that case, do you mind having the new options defaulted to the original settings in the env-template? (TACHYON_SSH_FOREGROUND and TACHYON_SLAVE_SLEEP)